examples/others/gridnav/lv_example_gridnav_4.py

```diff
@@ -0,0 +1,36 @@
+def event_handler(e):
+    obj = e.get_target()
+    list = obj.get_parent()
+    print("Clicked: " + list.get_btn_text(obj))
+
+
+#
+# Simple navigation on a list widget
```

Comment:
Somehow this doesn't look like the C example, and looses focus after clicking a file, or after clicking <kbd>TAB</kbd>